x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_upgrade_modal/index.tsx

```diff
@@ -132,7 +169,7 @@ export const AgentUpgradeAgentModal: React.FunctionComponent<Props> = ({
         {isSingleAgent ? (
           <FormattedMessage
             id="xpack.fleet.upgradeAgents.upgradeSingleDescription"
-            defaultMessage="This action upgrades the agent running on '{hostName}' to version {version}. You can't undo this upgrade."
+            defaultMessage="This action upgrades the agent running on '{hostName}' to version {version}. This action can not be undone. Are you sure you wish to continue?"
```

Comment:
can you change language to be "will upgrade" rather "upgrades"

Reply:
Sure thing. I assume we want the same approach for the other actions (assign, uenroll, etc)?

Reply:
updated in https://github.com/elastic/kibana/pull/96087/commits/606f3e64dac3e68a1eb2e64b793ffe24974b35c0